FaissMask/VectorTransform.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using FaissMask.Extensions;
+using FaissMask.Internal;
+
+namespace FaissMask
+{
+
+    public class VectorTransform(VectorTransformSafeHandle handle, int dimIn, int dimOut) : IDisposable
+    {
+        public static VectorTransform Read(string filename)
+        {
+            var handle = VectorTransformSafeHandle.Read(filename, ptr => new VectorTransformSafeHandle(ptr));
+            var dimIn = handle.DimensionIn;
+            var dimOut = handle.DimensionOut;
+            return new VectorTransform(handle, dimIn, dimOut);
+        }
+        
+        public int DimensionIn => handle.DimensionIn;
+        public int DimensionOut => handle.DimensionOut;
```

Comment:
Why are these delegating to the handle when the constructor for this object also takes the values? Seems like we should either use the constructor values, or get rid of them.